src/test/java/no/nav/pto/veilarbportefolje/tiltakshendelse/TiltakshendelseRepositoryTest.java

```diff
@@ -90,6 +90,52 @@ public void kanLagreFlereTiltakshendelserPaSammePerson() {
         assert (tiltakshendelser.size() == 2);
     }
 
+    @Test
+    void kanLagreSletteTiltakshendelse() {
+        UUID id = UUID.randomUUID();
+        LocalDateTime opprettet = LocalDateTime.now();
+        String tekst = "Forslag: endre varighet";
+        String lenke = "http.cat/200";
+        Fnr fnr = Fnr.of("11223312345");
+
+        KafkaTiltakshendelse hendelsePaEnPerson = new KafkaTiltakshendelse(id, true, opprettet, tekst, lenke, Tiltakstype.ARBFORB, fnr, Avsender.KOMET);
+
+        assertTrue(repository.tryLagreTiltakshendelseOgSjekkOmDenErEldst(hendelsePaEnPerson));
+
+        List<Tiltakshendelse> tiltakshendelser = repository.hentAlleTiltakshendelser();
+        assert (tiltakshendelser.size() == 1);
+
+        KafkaTiltakshendelse slettHendelsePaEnPerson = new KafkaTiltakshendelse(id, false, opprettet, tekst, lenke, Tiltakstype.ARBFORB, fnr, Avsender.KOMET);
+        repository.slettTiltakshendelseOgHentEldste(slettHendelsePaEnPerson.id(), slettHendelsePaEnPerson.fnr());
+
+        List<Tiltakshendelse> tiltakshendelserEtterInaktivMelding = repository.hentAlleTiltakshendelser();
+        assert (tiltakshendelserEtterInaktivMelding.isEmpty());
+    }
+
+    @Test
+    void kanLagreSletteTiltakshendelseReturnereNesteEldste() {
+        UUID id = UUID.randomUUID();
+        UUID idNyMelding = UUID.randomUUID();
+        LocalDateTime opprettet = LocalDateTime.now();
```

Comment:
Dei to meldingane burde ha ulike opprettet-tidspunkt for å sikre at dei ikkje er like gamle.

Dette har vi gløymd på dei andre testane også 😅